src/main/java/org/gitlab4j/api/PipelineApi.java

```diff
@@ -546,6 +546,11 @@ public PipelineSchedule takeOwnershipPipelineSchedule(Object projectIdOrPath, Lo
         return (response.readEntity(PipelineSchedule.class));
     }
 
+    public void playPipelineSchedule(Object projectIdOrPath, Long pipelineScheduleId) throws GitLabApiException {
+
+        post(Response.Status.CREATED, "", "projects", getProjectIdOrPath(projectIdOrPath),  "pipeline_schedules", pipelineScheduleId, "play");
```

Comment:
The empty String is not really necessary, right?

```suggestion
        post(Response.Status.CREATED, "projects", getProjectIdOrPath(projectIdOrPath),  "pipeline_schedules", pipelineScheduleId, "play");
```

Reply:
I changed it for the method above, too.

Thanks for the review! 😊

Reply:
Sorry I thought this is already part of the path…

But I double check with the other methods, and we never have `post(Response.Status.CREATED, "", ...)` in the code-base.

This seems to be the way other methods are doing it:

```suggestion
        post(Response.Status.CREATED, (Form)null, "projects", getProjectIdOrPath(projectIdOrPath),  "pipeline_schedules", pipelineScheduleId, "play");
```

Reply:
The method above (`takeOwnershipPipelineSchedule`) does it this way:

```        
Response response = post(Response.Status.OK, "", "projects", getProjectIdOrPath(projectIdOrPath),  "pipeline_schedules", pipelineScheduleId, "take_ownership");
```